src/routes/Home.js

```diff
@@ -216,6 +227,7 @@ export class Home extends Component {
             logs={logs}
             haveLogsLoaded={hasLoaded}
             checkboxList={checkboxList}
+            checkboxDisabled={isCheckboxesDisabled}
```

Comment:
And here it will look like this
```suggestion
            checkboxDisabled={isLogsDeletionInProgress}
```